src/components/BorrowerProfile/CommentsAndWhySpecial.vue

```diff
@@ -278,13 +343,34 @@ export default {
 				this.isLightboxVisible = false;
 				this.loading = false;
 			});
-		}
+		},
+		randomizedUserClass() {
+			const randomStyle = this.userCardStyleOptions[Math.floor(Math.random() * this.userCardStyleOptions.length)];
+			return `${randomStyle.color} ${randomStyle.bg}`;
+		},
+		determineIfMobile() {
+			this.isMobile = document.documentElement.clientWidth < 735;
+		},
+		isDefaultProfilePic(commentHash) {
+			if (!commentHash) {
+				return true;
+			}
+			const defaultProfileIds = [726677, 315726];
+			return defaultProfileIds.some(id => id.toString() === commentHash);
+		},
 	},
 	mounted() {
 		this.createObserver();
+		window.addEventListener('resize', _throttle(() => {
```

Comment:
The `_throttle` needs to be set to a component/unique method, otherwise `removeEventListener` won't remove the listener.